packages/bs-reform/__tests__/SchemaDemo_test.re

```diff
@@ -30,7 +30,9 @@ test("should validate a correct record", () => {
   module ProfileValidation = ReSchema.Make(ProfileLenses);
 
   let schema =
-    ProfileValidation.Validation.Schema([|Email(Email), IntMin(Age, 18)|]);
+    ProfileValidation.Validation.(
+      Schema(Belt.Array.concat(email(Email), int(~min=18, Age)))
```

Comment:
the local open of `Validation.(` already allows you to use the custom infix I provided just for that
```suggestion
      Schema(email(Email) + int(~min=18, Age))
```